src/osvr/Common/CMakeLists.txt

```diff
@@ -23,8 +23,11 @@ if(BUILD_WITH_TRACING)
     endif()
 endif()
 
+option(OSVR_COMMON_IN_PROCESS_IMAGING OFF)
```

Comment:
Think this is not quite the right syntax for `option()` - think there's usually a description in there.